### DISCUSSION TITLE
I made a new site

### DISCUSSION THREAD

**Discussion by Rplus** [Rplus/Pokemon-shiny#434] [General]
https://rplus.github.io/pokemongo-shiny/

There are simular UI with old (https://rplus.github.io/Pokemon-shiny/)
but different with data structure.

The new one will be able to add image source directly, instead of PokeMinor images

Help someone could give some feeback~ : P

**Comment by BartmanEH** [comment]
Wow. So much work! Thank you! I'll give it go.

**Comment by BartmanEH** [comment]
well it's pretty amazing. it also provides the functionality I was looking for in issue #52 -- by clicking on the buttons "Met" and "Own". Now, what does "Met" mean? What does "Much" mean?

**Reply by MelkyAir** [reply]
@Rplus i see "met" as registered, "much" as more than 1 in inventory and available for trade.

@Rplus love the new layout, works easy and also loads very fast!

**Reply by Rplus** [reply]
yes, I changed some words in the new version, but I'm not sure if it's appropriate for native English speakers.
`met`: you met that before, jsut like registered in older version, but `met` is shorter
`much`: just marked as tradeable, same reason, `much` is shorter

Maybe you could recommend other words for that~

**Reply by BartmanEH** [reply]
Maybe "reg'd" and "extra"?

**Reply by MelkyAir** [reply]
I would go for "Trade" instead of "much"

**Reply by BartmanEH** [reply]
well what one does with their "extra" is really up to them--maybe trade them... maybe transfer them to the professor when you need room :-)

**Comment by Rplus** [comment]
The data of new version will be excel table, but I use csv format in website.
https://github.com/Rplus/pokemongo-shiny/blob/main/src/assets/data/pm.csv

![image](https://github.com/user-attachments/assets/77657eb6-81a7-4e17-82ed-78d56d14b30b)

I think we can use google spreadsheet as the data source, and some collaborator could updte the sheet directly.
That means website could use latest data, instead waiting for me update that in github~

There are titles / categories in table:
https://github.com/Rplus/pokemongo-shiny/blob/main/README.md

**Reply by BartmanEH** [reply]
Thank you for your efforts! Considering you don't even play the game anymore, we sure do appreciate this.

**Reply by BartmanEH** [reply]
so with the new "tag" functionality, it seems we could add entirely new categories like "shadow", "dynamax", etc. right? And those of us that don't collect certain "tag" types, could just leave that filter turned off. Is your new software able to create filter buttons automatically from "detected" tags?

**Reply by Rplus** [reply]
yes, tags list is generated dynamically
but I think if user need shadow or dynamax, create new item for that is better, instead of just add a new tag for it

**Reply by BartmanEH** [reply]
"create new item".... not sure how you would do that
